packages/story-editor/src/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -175,9 +190,7 @@ function PageTemplatesPane(props) {
             <Select
               options={options}
               selectedValue={showDefaultTemplates ? DEFAULT : SAVED}
-              onMenuItemClick={(evt, value) =>
-                setShowDefaultTemplates(value === DEFAULT)
-              }
+              onMenuItemClick={(evt, value) => handleToggle(evt, value)}
```

Comment:
🤔 I usually prefer pure functions like `handleToggle` but in this case the `value` doesn't change between children or renders right? If it's pretty static - then I think it wouldn't be a problem to reference the `value` that's held in the component scope in the `handleToggle`

Further, that could save us a render because this inline arrow function is referentially different each render. Really just a nit, but this works great too.

Reply:
Duh! GP. GP.  I just left it when I copied and moved up, didn't even think about it. 🙌🏽

Reply:
Not a big deal 😄